ANALYSIS/ANALYSISalice/AliAnalysisTaskSE.cxx

```diff
@@ -57,6 +57,19 @@
 
 ClassImp(AliAnalysisTaskSE)
 
+namespace
+{
```

Comment:
Use three spaces indent (unfortunately this is the "standard" in most AliRoot classes)